ledfx_frontend/app/handlers.js

```
@@ -0,0 +1,137 @@
+
+  try {
+    switch (parameters.command) {
+      case 'close-others':
+        BrowserWindow.getAllWindows().forEach(win => {
+          if (win !== wind) {
+            win.close();
+          }
+        });
+        break;
+      case 'get-all-windows':
+        const allWIndows = BrowserWindow.getAllWindows()
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)